src/clarity/forms/_forms.clarity.scss

```diff
@@ -434,6 +433,23 @@ $clr-custom-checkbox-radio-top: ($clr_baselineRem_1 - $clr-checkbox-radio-height
             display: inline-block;
         }
 
+        input[type="checkbox"]:indeterminate + label::before {
```

Comment:
@hippee-lee: Just one small thing. If you are planning to add this demo to the website, we should use Clarity buttons